docs/Splashkit/Applications/Arcade Machines/Guide to Contribute a Game/Contributors Guide.md

```diff
@@ -0,0 +1,152 @@
+# Guide to Contribute a Game to the Arcade Machine 
+
+### This guide takes you through the steps required for your game to be added into the arcade-machine library.  
+---
+<br>
+Table of contents
+   
+1. 
+2. [Coding](#coding)
+   - [Quit Request](#quit-request)
+   - [Window Size](#window-size)
+   - [Window Border](#window-border)
+3. [Controls]()
+4. [Compiling]()
+5. [Artwork]()
+6. [Configuration]()
+7. [Content]()
+8. [Contributing]()
+
+---
+
+<br>
+
+## **Coding**
+To make the game accessible and controllable by the arcade machine, some additional code or changes are required
+
+<br>
+
+### *Quit Request*
+Your game must be able to be exited using the escape key
+This can be achieved by including the following command in your main loop. 
+
+```cpp
+int main()
+{
+    while(!key_down(ESCAPE_KEY))
+    {
+        // game play
+    }
+}
+```
+<br>
+
+### *Window Size*
+The window size of your game cannot exceed 1600 x 900, this is to allow your game to sit neatly inside the arcade-machine itself. Similarly, but mainly for aesthetic purposes, a minimum window size of 640 x 480 is expected. 
+
+<br>
+
+### *Window Border*
+Your game will look much better within the Arcade Machine environment if it doesn't show the window a border, so we ask that you remove the border before compiling your game.  
+To remove the border of your game window, use SplashKit’s `window_toggle_border();` function after the `open_window()` function like so:
+```cpp
+int main()
+{
+    open_window("my game", width, height);
+    window_toggle_border("my game");
+}
+```
+<br>
+
+## **Compiling**
+Only Windows compiled games are supported atm... 
+(Brief windows compilation explanation required here)
+
+<br>
+
+## **Artwork**
+A preview of your game will be shown in the Arcade Machine games menu so you will need to include an image of your game.  
+This image must be sized as 600px x 540px for it to be displayed correctly in the games menu.  
```

Comment:
```suggestion
A preview of your game will be shown in the Arcade Machine games menu. Please include an image of your game.  
This image must be sized as 600px x 540px so it will be displayed correctly in the games menu.  
```